code/game/turfs/open/floor/prison_floor.dm

```diff
@@ -0,0 +1,97 @@
+
+/turf/open/floor/prison
+	name = "secure floor"
+	desc = "Prison break-proof!"
+	icon = 'icons/turf/prisonfloor.dmi'
+	icon_state = "prisonfloor4"
+	max_integrity = 500
+	holodeck_compatible = TRUE
+	thermal_conductivity = 0.025
+	heat_capacity = INFINITY
+	floor_tile = /obj/item/stack/sheet/plasteel
+	footstep = FOOTSTEP_PLATING
+	barefootstep = FOOTSTEP_HARD_BAREFOOT
+	clawfootstep = FOOTSTEP_HARD_CLAW
+	heavyfootstep = FOOTSTEP_GENERIC_HEAVY
+	tiled_dirt = FALSE
+	var/plates_type = /obj/item/stack/tile/plasteel
+	var/plates = 4
+	var/wrenching = FALSE
+
+/turf/open/floor/prison/examine(mob/user)
+	. = ..()
+	. += "<span class='notice'>The reinforcement plates are <b>wrenched</b> firmly in place.</span>"
+
+/turf/open/floor/prison/break_tile()
+	return //unbreakable
+
+/turf/open/floor/prison/burn_tile() // consider changing this
+	return //unburnable
+
+/turf/open/floor/prison/make_plating(force = 0) // I don't know what that is
+	if(force)
+		..()
+	return //unplateable
+
+/turf/open/floor/prison/try_replace_tile(obj/item/stack/tile/T, mob/user, params)
+	return
+
+/turf/open/floor/prison/crowbar_act(mob/living/user, obj/item/I)
+	if(plates != 0)
+		to_chat(user, "<span class='danger'> The reinforcement plates are still firmly in place!</span>")
+		return
+	else
+		playsound(src, 'sound/machines/airlock_alien_prying.ogg', 100, TRUE)
+		to_chat(user, "<span class='notice'>You begin prying open the tile...</span>")
+		do_after(user, 4 SECONDS)
+		new floor_tile(src, 1)
+		ScrapeAway(flags = CHANGETURF_INHERIT_AIR)
+		. = ..()
```

Comment:
This will call pry_tile, which you probably don't want if you are overriding that behaviour. If you do want it (since it calls the crowbar sound), then make_plating should handle scraping away the tile instead and this should return ..() instead of calling scrape away.